app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicListViewModel.kt

```diff
@@ -31,4 +31,17 @@ class MusicListViewModel @Inject constructor(
     fun postMusic(): LiveData<List<MusicRepoModel>> {
         return postMusicList
     }
+
+    private fun saveMusic() {
+        viewModelScope.launch {
```

Comment:
in ja moshkel dareh.
vaghti dar hal save to databse hast foran getLocalMusic() run miseh dar natijeh list khali nesho mideh.
agar app kamel close konim (beh dalil lifcycle view model , fragment ) va open besh on mogheh list neshon mideh.